java-frontend/src/main/java/org/sonar/plugins/java/api/tree/ClassTree.java

```diff
@@ -86,6 +86,14 @@
 
   ListTree<TypeTree> superInterfaces();
 
+  /**
+   * @since Java 15
+   * @deprecated Preview Feature
+   */
+  @Deprecated
+  @Nullable
+  SyntaxToken permitsKeyword();
```

Comment:
Should we document this change in the [documentation: API changes](https://github.com/SonarSource/sonar-java/blob/master/sonar-java-plugin/src/main/resources/static/documentation.md)?

Reply:
Nop, we do not document changes related to preview features yet.